base/applications/network/telnet/src/tnconfig.cpp

```diff
@@ -547,7 +547,7 @@ void TConfig::keyfile_init() {
 		// if there is no environment variable
 		GetPrivateProfileString("Keyboard", "Keyfile", "", keyfile,
 			sizeof(keyfile), inifile);
-		if(keyfile == 0 || *keyfile == 0) {
+		if(keyfile == NULL || *keyfile == 0) {
```

Comment:
Note that this first `keyfile == 0` (or NULL) doesn't make sense, because `keyfile` is a fixed buffer: `char keyfile[MAX_PATH*2];` member of the `TConfig` class. If it were a pointer on the other hand, it would be OK (but then, these `sizeof(keyfile)` would be wrong...)